charts/firefly/templates/evmconnect/secret.yaml

```diff
@@ -0,0 +1,80 @@
+{{/*
+  Copyright © 2022 Kaleido, Inc.
+
+  SPDX-License-Identifier: Apache-2.0
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://swww.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/}}
+
+{{- if .Values.evmconnect.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "firefly.fullname" . }}-evmconnect-config
+  labels:
+    {{- include "firefly.evmconnectLabels" . | nindent 4 }}
+stringData:
+  config.yaml: |-
+    {{- with .Values.evmconnect.config.confirmations }}
+    confirmations:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+
+    {{- with .Values.evmconnect.config.eventstreams }}
+    eventstreams:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+
+    {{- with .Values.evmconnect.config.webhooks }}
+    webhooks:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+
+    {{- with .Values.evmconnect.config.policyLoop }}
+    policyloop:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+
+    {{- with .Values.evmconnect.config.transactions }}
+    transactions:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+
+    {{- tpl .Values.evmconnect.config.policyEngine . | nindent 4 }}
```

Comment:
Note that a policy engine is required, but currently only a simple engine has been implemented in the open source.

This template by default configures the simple engine, but offers the ability to customize its config and / or plugin future policy engines.